packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -163,73 +163,125 @@ This is prototyped in the [formatted-text-column branch](https://github.com/ni/n
+
+This column will trigger `invalidColumnConfiguration` on the table's validity state and will include flags in the column's validity state if its configuration can't be translated to a valid `Intl.DateTimeFormat` object. To provide better developer feedback about what's wrong with the configuration, the column could expose a public method like `createDateTimeFormat()` which would use the column's configuration to construct the formatter but allow any exceptions to be thrown.
+
+##### Examples
+
+```html
+<nimble-table>
+    <nimble-table-column-date-text field-name="start" placeholder="Not started">
+        Start
+    </nimble-table-column-date-text>
+
+    <nimble-table-column-date-text
+        field-name="dueDate"
+        format="custom"
+        custom-date-style="full"
+    >
+        Due Date
+    </nimble-table-column-date-text>
+</nimble-table>
+```
 
-_*Props/Attrs*_
+#### Elapsed Time Column
 
-_Type Reference_
+Nimble could introduce a column type that maps time duration to localized strings (e.g. "1 minute, 10 seconds"). Its API will be discussed in a future update to this document. It might use [Temporal duration values](https://tc39.es/proposal-temporal/docs/duration.html) formatted with [`Intl.DurationFormat`](https://tc39.es/proposal-intl-duration-format/) but those APIs are not yet approved for implementation in browsers. The [`Intl.RelativeTimeFormat` API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat) also looks promising but it includes relative words like "ago" and "in" which are difficult to remove, even when using [`formatToParts`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat/formatToParts).
 
-### Anatomy
+#### Specifying locale
 
-### Angular integration
+The columns described above will format numbers, dates, and times differently depending on the locale. To specify the locale, the column implementations will read a Nimble configuration design token for locale and pass its value to the `locales` input of the `Intl.NumberFormat` or `Intl.DateTimeFormat` constructor. If the value of that token changes on the column element, the column will re-render with the new locale.
 
-### Blazor integration
+The design token for configuring locale will be a new token following the patterns described in [Automatic theme / Runtime Configuration HLD](https://github.com/ni/nimble/pull/1257) for tokens like `theme` and `direction`. Its attribute name on the theme provider will be `locale`. It will be a string expected to conform with the [BCP 47 locale format](https://en.wikipedia.org/wiki/IETF_language_tag) (e.g. `en-us` or `de`). Its initial value will be set by reading the `lang` attribute from the page's `<html>` element.
```

Comment:
I attempted to capture some of this in https://github.com/ni/nimble/issues/1293 since similar ideas had come up as future work for string column sorting. Feel free to update if I didn't capture everything completely or correctly.